pkg/systemd/logs.js

```diff
@@ -103,7 +104,7 @@ $(function() {
         ReactDOM.render(
             React.createElement(EmptyStatePanel, {
                 loading: loading,
-                showIcon: show_icon,
```

Comment:
Right, this commit is not ideal.
The `showIcon` was there so we could have only `Loading...` without any icons (nor spinner, nor ExclamationCircle). This is happening when you scroll to the bottom of logs and click on `Load earlier entries`. We don't want to show spinner there.

Reply:
To clarify: On 213, when I click on "Load earlier entries..", I *only* get a spinner, no "Loading..." text. That's indeed a change in this PR, where you get a spinner *and* the "Loading..." text. I'm happy to change that, but I have the feeling that's not what you meant. Let's discuss on IRC for faster bandwidth?

Reply:
I was wrong -- master/213 actually does show "Loading.." as well, it was just scrolled off the screen. So the original PR kept that behaviour. As per IRC conversation, I now added a separate commit to hide the "Loading..." when logs are present.